### ISSUE TITLE
Add the dimension of historical cartography

### ISSUE THREAD

**Issue by warrenjp** [ohmec/ohmec#77]
Might be interesting to show how the Western World's understanding of North and South American evolved over time as more explorers reported back with their findings. Not only did internal borders change, but the external shapes of coastlines changed over time.

e.g. 
https://www.loc.gov/collections/discovery-and-exploration/articles-and-essays/the-1562-map-of-america/

This is a very Western view of the Americas since, of course, the indigenous peoples were well aware of the geography of the coastlines. Nevertheless, adding this additional dimension as a viewing option might be of interest and certainly would capture the uncertainty and guesswork characterizing the early Western explorers.

**Comment by ohmec** [comment]
Interesting concept.

> Not only did internal borders change, but the external shapes of coastlines changed over time.

Did you mean the Western World's conception of the coastlines changed (via better and better mapping technology) or coastlines truly did change (via earth / human events)? Both are interesting issues to visualize, not sure what the right answer is. The `fidelity` property can give indication of the confidence of the borders, but doesn't really have a way of addressing this, which would require changes to the fundamental feature coordinates.

**Comment by warrenjp** [comment]
> Did you mean the Western World's conception of the coastlines changed (via better and better mapping technology) or coastlines truly did change (via earth / human events)? 

The former. For example, early explorers thought that Baja California (and, frankly, all of California) was an island rather than peninsula. Thus, their early maps depicted Baja as such:

<img width="1199" alt="Screen Shot 2021-09-05 at 3 04 02 PM" src="https://user-images.githubusercontent.com/4216677/132142594-2577d82a-48ab-4948-9818-1484c4eaa601.png">

https://www.1stdibs.com/art/prints-works-on-paper/landscape-prints-works-on-paper/heinrich-scherer-america-borealis-california-as-island/id-a_8527962/

**Comment by ohmec** [comment]
Got it, that would be very cool. LMK when you've got a mock up 🤣 

More seriously, though: it does overly emphasize the western viewpoint, which is already a concern given the notion of "political entities". Perhaps an idea of a "explorer's lens" would be pretty interesting.

**Comment by warrenjp** [comment]
> More seriously, though: it does overly emphasize the western viewpoint, which is already a concern given the notion of "political entities". Perhaps an idea of a "explorer's lens" would be pretty interesting.

Precisely. Thus, I set this as a P3 feature and I'm going to actively work on adding in the Native American peoples now.

**Comment by ohmec** [comment]
Speaking of lenses, I've often thought of other things that are candidates: wartime perspectives (think: world war 2 expansion of germany, japan, etc); and empires (think: roman empire, with display of current emperor in an info box).
